app/controllers/api/v1/registered_urls_controller.rb

```diff
@@ -5,6 +5,15 @@ def index
         @registered_urls = session.registered_urls.active.not_expired
       end
 
+      def show
```

Comment:
If you did the change above in the `ProtectedApplicationController` because you need to unprotect this action, you can add in this controller this instead:
```rb
skip_before_action :authenticate_request, only: [:show]
```

This will be more specific than the approach you took on the other controller

Reply:
nice, thanks!